cdap-ui/app/cdap/components/AbstractWidget/KeyValueDropdownWidget/KeyValueDropdownRow.tsx

```diff
@@ -64,6 +71,7 @@ class KeyValueDropdownRow extends AbstractRow<IKeyValueDropdownRowProps, IKeyVal
   public static defaultProps = {
```

Comment:
We need to have a default value for ordering which should be `OrderingEnum.KEYSFIRST`